lib/decompress/zstd_decompress_block.c

```diff
@@ -715,7 +715,7 @@ size_t ZSTD_execSequence(BYTE* op,
 
     /* Errors and uncommon cases handled here. */
     assert(oLitEnd < oMatchEnd);
-    if (iLitEnd > litLimit || oMatchEnd > oend_w)
+    if (UNLIKELY(iLitEnd > litLimit || oMatchEnd > oend_w))
```

Comment:
All the LIKELY() and UNLIKELY() I've added actually make a difference in perf. Normally neutral for gcc and helpful for clang.